client/core/trade_simnet_test.go

```diff
@@ -543,7 +543,7 @@ func monitorTrackedTrade(ctx context.Context, client *tClient, tracker *trackedT
 			}
 
 			if assetToMine != nil {
-				assetID, nBlocks := assetToMine.ID, uint16(assetToMine.SwapConf)+1 // mine 1 extra to ensure tx gets req. confs
+				assetID, nBlocks := assetToMine.ID, uint16(assetToMine.SwapConf)
```

Comment:
No, but the extra blocks were added in (pretty sure) because we were grappling with issues getting txns mined on simnet, then we figured out we needed the regentemplate RPC in mine-alpha, which seems to have entirely resolved the mining issue.  Do you think we still need an extra block in any of these cases? The appropriate tests would only mine the blocks that reach inaction (just SwapConf).